components/resources/brave_components_strings.grd

```diff
@@ -717,7 +712,8 @@
       <message name="IDS_BRAVE_UI_TOKEN_GRANTS" desc="">Token Grants</message>
       <message name="IDS_BRAVE_UI_POINT_GRANTS" desc="">Point Grants</message>
       <message name="IDS_BRAVE_UI_TRANSACTIONS" desc="">Transactions</message>
-      <message name="IDS_BRAVE_UI_TURN_ON_ADS" desc="">Turn on Brave Ads</message>
+      <message name="IDS_BRAVE_UI_TURN_ON_ADS" desc="">I'm ready to Earn &amp; Give</message>
+      <message name="IDS_BRAVE_UI_TURN_ON_TEXT" desc="">Earn tokens by viewing privacy-respected ads and support your favourite web content creators by default.</message>
```

Comment:
Nit: Should "privacy-respected" be "privacy-respecting"? Also, "favourite" is the UK variant with the 'u', if that matters.

Reply:
fixed